commit e02de88e4e1f6ee155421311d7bc8cf7cc69f37a
Author: Maximilian Fridrich <m.fridrich@commend.com>
Date:   Fri Feb 28 08:43:44 2025 +0100

    Revert "res_rtp_asterisk.c: Set Mark on rtp when timestamp skew is too big"
    
    This reverts commit f30ad96b3f467739c38ff415e80bffc4afff1da7.
    
    The original change was not RFC compliant and caused issues because it
    set the RTP marker bit in cases when it shouldn't be set. See the
    linked issue #1135 for a detailed explanation.
    
    Fixes: #1135.

Index: asterisk-22.2.0/res/res_rtp_asterisk.c
===================================================================
--- asterisk-22.2.0.orig/res/res_rtp_asterisk.c
+++ asterisk-22.2.0/res/res_rtp_asterisk.c
@@ -5268,11 +5268,6 @@ static int rtp_raw_write(struct ast_rtp_
 	}
 
 	if (ast_test_flag(frame, AST_FRFLAG_HAS_TIMING_INFO)) {
-		if (abs(frame->ts * rate - (int)rtp->lastts) > MAX_TIMESTAMP_SKEW) {
-			ast_verbose("(%p) RTP audio difference is %d set mark\n",
-				instance, abs(frame->ts * rate - (int)rtp->lastts));
-			mark = 1;
-		}
 		rtp->lastts = frame->ts * rate;
 	}
 
